docs/src/notebooks/9_max_min_driving_force_analysis.jl

```diff
@@ -97,26 +97,30 @@ reaction_standard_gibbs_free_energies = Dict( # kJ/mol
     "GLUt2r" => -3.49,
     "PPS" => -6.05,
     "FUM" => -3.42,
-)
-
-# In general you cannot be certain that all fluxes will be positive. This poses problems
-# for systematically enforcing that ΔᵣG ≤ 0 as done in the original formulation. In COBREXA
-# ΔᵣG ⋅ vᵢ ≤ 0 is instead enforced, where vᵢ is the flux of reaction i. By default all fluxes
-# are assumed to be positive, but by supplying thermodynamically consistent flux solution
-# it is possible to drop this implicit assumption and makes it easier to directly incorporate
-# the max min driving force into non-customized models.
+);
 
-flux_solution = flux_balance_analysis_dict(
+# In general you cannot be certain that all fluxes will be positive for a given flux
+# solution. This poses problems for systematically enforcing that ΔᵣG ≤ 0 for each reaction,
+# because it implicitly assumes that all fluxes are positive, as done in the original
+# formulation of MMDF. In COBREXA we instead enforce ΔᵣG ⋅ vᵢ ≤ 0, where vᵢ is the flux of
+# reaction i. By default all fluxes are assumed to be positive, but by supplying
+# thermodynamically consistent flux solution it is possible to drop this implicit assumption
+# and makes it easier to directly incorporate the max min driving force into non-customized
+# models. Here, customized model means a model written such that a negative ΔᵣG is associated
+# with each positive flux in the model, and only positive fluxes are used by the model.
+
+flux_solution = flux_balance_analysis_dict( # find a thermodynamically consistent solution
     model,
     GLPK.Optimizer;
     modifications = [add_loopless_constraints()],
 )
 
-# Run max min driving force analysis with some reasonable constraints. Protons and water are
-# removed from the concentration calculation of the optimization problem, thus we specify
-# their IDs explicitly.  The reason for this is that the Gibbs free energies of biochemical
-# reactions is measured at constant pH, so proton concentrations is fixed; likewise we
-# assume that reactions occur in aqueous environments, hence water is excluded too.
+# Run max min driving force analysis with some reasonable constraints on metabolite
+# concentration bounds. Note, protons and water are removed from the concentration
```

Comment:
To remove protons and water from the concentration calculations, we explicitly specify their IDs.